.github/actions/ipsec/configs.yaml

```diff
@@ -0,0 +1,86 @@
+- name: '1'
+  # renovate: datasource=docker depName=quay.io/lvh-images/kind
+  kernel: 'rhel8-20240404.144247'
+  kube-proxy: 'iptables'
```

Comment:
Note that this file is currently not in scope of our renovate config, so we'll no longer auto-bump them.

Reply:
Thanks! Fix is at https://github.com/cilium/cilium/pull/35555.